doc/getting_started.md

```diff
@@ -113,6 +113,8 @@ use the name of your module as the name for your new repository.
     git remote add origin https://github.com/[GITHUB_USER_OR_ORG_NAME]/[GITHUB_MODULE_NAME].git
     git push -u origin master
 
-## 9) Register your method thorugh the Narrative
+## 9) Register your method through the Narrative
+
+Documentation for this is available in the Quick Start section of toplevel SDK documentation.
```

Comment:
I offhand don’t know the markdown magic for relative links, and it doesn’t make sense to have an absolute link since this is in a downloaded document.

On Dec 18, 2015, at 3:53 PM, Nomi Harris <notifications@github.com<mailto:notifications@github.com>> wrote:


In doc/getting_started.md<https://github.com/kbase/kb_sdk/pull/66#discussion_r48074227>:

> @@ -113,6 +113,8 @@ use the name of your module as the name for your new repository.
>      git remote add origin https://github.com/[GITHUB_USER_OR_ORG_NAME]/[GITHUB_MODULE_NAME].git
>      git push -u origin master
>
> -## 9) Register your method thorugh the Narrative
> +## 9) Register your method through the Narrative
> +
> +Documentation for this is available in the Quick Start section of toplevel SDK documentation.


"toplevel" -> the top level
How about adding a link to that? https://github.com/kbase/kb_sdk/blob/master/README.md

—
Reply to this email directly or view it on GitHub<https://github.com/kbase/kb_sdk/pull/66/files#r48074227>.